crates/cast/src/main.rs

```diff
@@ -321,6 +350,7 @@ async fn main() -> Result<()> {
             print_command_result("show-config", &mut result, value_format, cli.json)?;
             Ok(())
         }
+        Commands::Script(_) => Err(anyhow!("All cases should already be handled")),
```

Comment:
Is this what you mean?
```suggestion
        Commands::Script(_) => unreachable!(),
```